config/betterquesting/DefaultQuests/Quests/EndgameGoals-AAAAAAAAAAAAAAAAAAAAHQ==/Eternity-wGq5FvuuROyyMPbT_SkMYg==.json

```diff
@@ -29,7 +37,7 @@
       "lockedProgress:1": 0,
       "autoClaim:1": 0,
       "isSilent:1": 0,
-      "desc:8": "Definitely consider this for the better Magnetohydrodynamically Constrained Star Matter recipe.\n\nKickstarting the loop is expensive and RNG based, you need 20,736L of Universium for the first 5 attempts and a catalyst.\n\nThe QFT recipe has a 25% chance of yielding Eternity per attempt, you should probably boost the QFT with the plasmas, and focus Eternity.\n\n[note]Did you know that the QFT has a fluid mode? Could be useful for getting molten shirabon out of it.[/note]\n\n[note]Building a full beacon out of Eternity Blocks would be quite silly, right?[/note]\n\n"
+      "desc:8": "Definitely consider this for the improved Magnetohydrodynamically Constrained Star Matter recipe. You’ll also need it to make MHDCSM parts.\n\nKickstarting the loop is expensive and RNG-based. You need 20,736 L of Universium for the first 5 attempts and a catalyst.\n\nThe QFT recipe has a 25% chance of yielding Eternity per attempt. You should probably boost the QFT with plasmas and focus on producing Eternity.\n\n[note]Did you know the QFT has a fluid mode? It could be useful for extracting molten Shirabon.[/note]\n\n[note]Building a full beacon out of Eternity Blocks might be a bit extravagant, don’t you think?[/note]\n\n"
```

Comment:
Would definitely mention *how* to toggle the QFT fluid mode instead of only mentioning that it exists

Reply:
resolved